chromium-urlbar-colon-alash-20200219.json

```diff
@@ -0,0 +1,15 @@
+{
+  "area": [
+    {
+      "xpos": 207,
+      "ypos": 70,
+      "width": 12,
+      "height": 19,
+      "type": "match"
+    }
+  ],
+  "properties": [],
+  "tags": [
+    "chromium-urlbar-colon-alash"

```

Comment:
This should be actually called **chromium-hightlighted-urlbar**

Reply:
updated: http://artemis.suse.de/tests/2266#step/chromium/19